gradoop-flink/src/main/java/org/gradoop/flink/datagen/transactions/foodbroker/config/FoodBrokerConfig.java

```diff
@@ -87,14 +89,16 @@ public static FoodBrokerConfig fromJSONString(String configString)
    * @return list of String
    */
   public List<String> getStringValuesFromFile(String fileName) {
-    List<String> values = null;
+    String value = "";
+
+    InputStream inputStream = this.getClass().getResourceAsStream("/foodbroker/" + fileName);
     try {
-      values = FileUtils.readLines(FileUtils.getFile(
-        FoodBrokerConfig.class.getResource("/foodbroker/" + fileName).getFile()));
+      value = org.apache.commons.io.IOUtils.toString(inputStream, "UTF-8");
```

Comment:
What if the file does not exist? This line will throw a NPE in this case, breaking the rest of the code.